docs/Contributing/Configuration-for-contributors.md

```diff
@@ -216,6 +216,64 @@ Whether the integration is configured to create Zendesk tickets for recent softw
         enable_software_vulnerabilities: true
   ```
 
+## Setup for macOS hosts that automatically enroll
+
+Automatic enrollment is an MDM feature available in Fleet Premium.
+
+These configuration options were built for a proof of concept for a prospective Fleet customer.
+
+### Okta
+
+#### mdm.okta_server_url
+
+This is the URL of your Okta [authorization server](https://developer.okta.com/docs/concepts/auth-servers/)
+
+- Default value: ""
+- Environment variable: `FLEET_MDM_OKTA_SERVER_URL`
+- Config file format:
+  ```
+  mdm:
+    okta_server_url: https://example.okta.com
+  ```
+
+#### mdm.okta_client_id
+
+This is the client ID of the Okta application that will be used to authenticate users. This value can be found in the Okta admin page under "Applications > Client Credentials."
+
+- Default value: ""
+- Environment variable: `FLEET_MDM_OKTA_CLIENT_ID`
+- Config file format:
+  ```
+  mdm:
+    okta_client_id: 9oa4eoxample2rpdi1087
+  ```
+
+#### mdm.okta_client_secret
+
+This is the client secret of the Okta application that will be used to authenticate users. This value can be found in the Okta admin page under "Applications > Client Credentials."
+
+- Default value: ""
+- Environment variable: `FLEET_MDM_OKTA_CLIENT_SECRET`
+- Config file format:
+  ```
+  mdm:
+    okta_client_secret: COp8o5zskEQ0OylgjqTrd0xu7rQLx-VteaQW4YGf
+  ```
+
+### End user license agreement (EULA)
+
+#### mdm.eula_url
+
+An URL containing a PDF file that will be used as an EULA during DEP onboarding.
```

Comment:
Is EULA provided as part of SAML? If not, then we still need this right?

Reply:
Yes, it is. However, for SAML, the URL will be provided in the `config` YAML applied via `fleetctl`. These are docs for Fleet server configuration (not for `fleetctl`)